parsing/asttypes.mli

```diff
@@ -25,6 +25,7 @@ type rec_flag = Nonrecursive | Recursive
 
 type direction_flag = Upto | Downto
 
+(* Order matters, used in polymorphic comparison *)
```

Comment:
I don't remember other occurrences, but I worked on that one year and a half ago, I more than likely forgot details.